### DISCUSSION TITLE
Is there a recommended way to configure different external contract addresses depending on deployment?

### DISCUSSION THREAD

**Discussion by alvatar** [scaffold-eth/scaffold-eth-2#376] [Q&A] [answered]
Say I want to deploy on Goerli, there the addresses are different than Mainnet.
Is there a canonical or recommended way to handle this, for example so that the `yarn build` command is build against one or another network/environment, and as such it picks some addresses and not others?

**Comment by sverps** [comment]
By setting the target network in `scaffold.config.ts`, the project resolves the correct `address` from the `contract.ts` file.

**Reply by alvatar** [reply, marked as answer]
Thanks! I think the answer is related to local contracts (defined in the project), whereas I was asking about external ones.
But I think I got the answer: the `networks.ts` file had the `NETWORKS_EXTRA_DATA` variable that I can use to add these external addresses.
